index.html

```diff
@@ -16,61 +26,49 @@
+              </div>               
               </div>
-              <div class="Wifi">
-              <img src="./images/Wifi-path.png" class="Wifi-path-2" alt="wifi">
-              <img src="./images/Wifi-path.png" class="Wifi-path-1" alt="wifi">
-              <img src="./images/Wifi-path.png" class="Wifi-path" alt="wifi">
+              <div class="Wii">
+                <i class="fa fa-wii"></i>
               </div>
             </div>  
           </div>
           </div>  
           <div id="Toolbar-Dark-Compact">
             <div class="logoPlaceholder">
-              <a href="#"><img src="./images/My Logo.png" class="myLogo" alt="Logo"></a>
+              <a href="#"><img src="./images/myLogo.png" class="myLogo" alt="Logo"></a>
             </div>
             <div class="Normal-Button-Tertiary-Enabled">
-                <div id="icons-icons-menu">
-                  <li><a href="#Portfolio">Portfolio</a></li>
-                  <li><a href="#Portfolio">Portfolio</a></li>
-                    <a href="#About">About</a>
-                    <a href="#Contact">contact</a>
-                  </div>
-                  <a href="javascript:void(0);" class="Union" onclick="myFunction()" >
-                    <img src="./images/Vector2.png" class="Vector-2" alt="">
-                    <img src="./images/Vector2.png" class="Vector-1" alt="">
-                    <img src="./images/Vector2.png" class="Vector" alt="">
-                  </a>
-                  
-                </div>
-              
+              <div id="myLink">
+                <ul>
+                  <li><a href="#">Portfolio</a></li>
+                  <li><a href="#About">About</a></li>
+                  <li><a href="#Contact">Contact</a></li>
+                </ul>
+              </div>
+              <a href="javascript:void(0);" class="icon" onclick="menuBar()">
+              <i class="fa fa-bars"></i>              
+              </a>
+            </div>
           </div>
         </nav>
       </header>
       <main class="header-Shapes mobile">
-        <img class="Rectangle-4" src="./images/Rectangle 4.png">
-        <img class="Ellipse-3" src="./images/Ellipse 3.png">
-        <img class="Ellipse-15" src="./images/Ellipse 15.png">
-        <img class="Ellipse-14" src="./images/Ellipse 14.png">
-        <img class="Ellipse-8" src="./images/Ellipse 8.png">
-        <img class="Ellipse-11" src="./images/Ellipse 11.png">
-        <img class="Rectangle-6" src="./images/Rectangle 6.png">
-        <img class="Ellipse-12" src="./images/Ellipse 12.png">
-        <img class="Ellipse-13" src="./images/Ellipse 13.png">
-        <img class="Ellipse-4" src="./images/Ellipse 4.png">
-        <img class="Rectangle-5" src="./images/Rectangle 5.png">
-        <img class="Rectangle-7" src="./images/Rectangle 7.png">
-        <img class="Ellipse-5" src="./images/Ellipse 5.png">
+        <img src="./images/header-shapes mobile.png">
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you use `background-image` css property for your image because this image is only for decorative purposes.